CHANGELOG.md

```diff
@@ -5,22 +5,24 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.2.0] - 2020-11-11
+
  * [`changed`] `base_config.inc` to work with I2C and SHDLC at the same time.
                Changed HAL implemantation locations and moved the UART one from
                the Makefile of the test in the sensor repo to here. Also changed
-               the way how mux sources are included so they are only needed by i2c
+               the way how mux sources are included so they are only needed by I2C
                tests.
- * [`added`] SHDLC implementation. This implementation was immigrated from the
-             `embedded-uart-sps` [repository][embedded-uart-common]. Since it's
-             general code which could be used by multiple sensors it makes more
-             sense to interate it here. The Shdlc part also uses the files in
-             `common/`. In addition to that it adds two `.[ch]` file pairs.
-             First `sensirion_shdlc.[ch]` which contains the SHDLC implementation
-             for Sensirion sensors. Sencondly `sensirion_uart_hal.h` and
-             `sensirion_uart_implementation.c` which describe the hardware
-             abstraction layer used by `sensirion_shdlc.c`.
-             This hal can be implemented in `sensirion_uart_implentation.c` for
-             the platform the sensor will be used on.
+ * [`added`]   SHDLC implementation. This implementation was immigrated from the
```

Comment:
I just now noticed the word "immigrated" 😆 never heard it in this context before. Maybe better use "imported" or "This implementation is originally from"

Reply:
Do you have something against immigration? Are you racist against poor code from embedded-uart-sps? ;)
It's changed :+1: